.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+# Azure IoTEdge PR checklist:
+
+This checklist is used to make sure that common guidelines for a pull request are followed.
+
+### General Guidelines and Best Practices
+- [ ] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
+- [ ] Title of the pull request is clear and informative.
+- [ ] Description of the pull request includes a brief summary of the enhancement or bug fix.
+- [ ] There are a small number of commits, each with an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
+
+### Testing Guidelines
+- [ ] Description of the pull request includes a brief summary of tests added/modified and the local testing done.  
```

Comment:
I think this should be two separate items with their own checkboxes (1. brief summary of tests added/modified, 2, local testing done)